### PR TITLE
Raees/user sessions

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@marykhan

File: user_sessions/urls.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+
+app_name = 'user_sessions'
+urlpatterns = [
+    url(r'^$', views.HomePageView.as_view(), name='home'),
```
Comment (MaryKhan) [inline user_sessions/urls.py]: why HomePageView n AddUserView urls are in this module?

File: user_sessions/views.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class LogoutView(View):
+
+    def get(self, request):
+        logout(request)
+        return HttpResponseRedirect('online_store:home')
+
+
+class AddUserView(View):
```
Comment (MaryKhan) [inline user_sessions/views.py]: move this view (along with template) in users app.
also think about HomePageView.

File: user_sessions/views.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    template_name = 'user_sessions/registration.html'
+    form_class = UserRegistrationForm
+
+    def get(self, request):
+        form = self.form_class
```
Comment (MaryKhan) [inline user_sessions/views.py]: we don't need re-assignment.

File: user_sessions/forms.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label='Username', max_length=50)
+    password = forms.CharField(widget=forms.PasswordInput())
+
+
+class UserRegistrationForm(UserCreationForm):
```
Comment (MaryKhan) [inline user_sessions/forms.py]: same deal!
move this to the users app.

**Review by MaryKhan — CHANGES_REQUESTED** [review]

**Comment by raeeschachar** [comment]
@MaryKhan Done. Please review.

**Comment by MaryKhan** [comment]
good to merge!

**Pull request merged**